app/components/ReproChecklists/ChecklistItem.js

```diff
@@ -0,0 +1,163 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import styles from './ChecklistItem.css';
+import NoteEditor from '../NoteEditor/NoteEditor';
+
+function ChecklistItem({ item, project, onUpdatedNote, onDeletedNote, onAddedNote }) {
+  const [answer, setAnswer] = useState(item.answer);
+  const [notes, setNotes] = useState(item.userNotes || []);
+
+  const handleNoteUpdate = (note, text) => {
+    if (note) {
+      if (onUpdatedNote) {
+        onUpdatedNote(project, text, note);
+      }
+    } else {
+      if (onAddedNote) {
+        onAddedNote(project, text);
+      }
+    }
+  };
+
+  const handleNoteDelete = (note) => {
+    if (onDeletedNote) {
+      onDeletedNote(project, note);
+    }
+  };
+
+  useEffect(() => {
+    setNotes(item.userNotes || []);
+  }, [item.userNotes]);
+
+  return (
+    <div className={styles.item}>
+      <div className={styles.header}>
+        <span>{item.statement}</span>
+        <div className={styles.buttoncontainer}>
+          <button
+            className={answer === 'yes' ? styles.yesset : styles.yes}
+            onClick={() => setAnswer('yes')}
+          >
+            {'Yes'}
+          </button>
+          <button
+            className={answer === 'no' ? styles.noset : styles.no}
+            onClick={() => setAnswer('no')}
+          >
+            {'No'}
+          </button>
+        </div>
+      </div>
+      <div className={styles.details}>
+        {notes.length > 0 && (
+          <div className={styles.notes}>
+            <h4>Notes:</h4>
+            <ul>
```

Comment:
The `NoteEditor` control will show existing notes, so I don't know that we need to display them again here.  Consider removing.

Reply:
oh yes! i was actually thinking of passing an empty array to the `NoteEditor`, but ya this works btr!